packages/react-components/react-button/library/src/components/Button/useButtonStyles.styles.ts

```diff
@@ -35,23 +35,23 @@ const useRootBaseClassName = makeResetStyles({
   margin: 0,
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31692&commitId=78d85730508970093b4f531b296a7355e3652bad&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Removed|
|SwatchPicker Converged.default - Dark Mode.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>